app/scripts/modules/ecs/placementStrategy/placementStrategy.service.ts

```diff
@@ -0,0 +1,51 @@
+import { module } from 'angular';
+
+import { PlacementStrategy } from './PlacementStrategy';
+
+export class PlacementStrategyService {
+
+  public getPredefinedStrategy(strategyName: string): PlacementStrategy[] {
+    if ( strategyName === 'AZ Balanced Spread' ) {
+      return this.getAzBalancedSpreadStrategy();
+    } else if ( strategyName === 'AZ Balanced BinPack' ) {
+      return this.getAzBalancedBinPackStrategy();
+    } else if ( strategyName === 'BinPack' ) {
+      return this.getBinPackStrategy();
+    } else if ( strategyName === 'One Task Per Host' ) {
+      return this.getOneTaskPerHostStrategy();
+    } else {
```

Comment:
You could add a TODO that indicates that custom needs to be implemented as well.  but that's super optional

Reply:
Done.